packages/wonder-blocks-form/src/components/text-field.js

```diff
@@ -183,6 +190,18 @@ export default class TextField extends React.Component<Props, State> {
             required,
             light,
             style,
+            testId,
+            // The following props are being included here to avoid
+            // passing them down to the otherProps spread
+            /* eslint-disable no-unused-vars */
+            onFocus,
+            onBlur,
+            onValidation,
+            validation,
+            onChange,
+            /* eslint-enable no-unused-vars */
+            // Should only include Aria related props
+            ...otherProps
```

Comment:
Do y'all know a better way to have the `otherProps` spread only contain the props related to the `AriaProps` type?

Reply:
nope, I cannot find of a better way for doing that.